tests/xfstests/install.pm

```diff
@@ -26,6 +26,16 @@ use repo_tools 'add_qa_head_repo';
 
 my $STATUS_LOG  = '/opt/status.log';
 my $VERSION_LOG = '/opt/version.log';
+my $LOCAL_DIR   = '/tmp/xfstests-dev';
+my $IBS_REPO    = get_var('XFSTEST_IBS_REPO', 'http://download.suse.de/ibs/home:/yosun:/branches:/QA:/Head/SLE-15-SP2');
+
+sub install_xfstests_from_ibs {
+    add_qa_head_repo;
+    zypper_call("ar $IBS_REPO ibs");
+    zypper_call('--gpg-auto-import-keys ref');
+    zypper_call('in xfstests');
+    zypper_call('in fio');
+}
```

Comment:
You can use xfstests from QA:Head now (added latest version from 20200420, just wait till it is builded ).

Add repo with lower priority like it is done in blktests.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/617cced3a908e9c11e4bc63b51f5ba1fdb3ccd12/tests/kernel/blktests.pm#L45

Reply:
Agree, solved.